templates/wizard.php

```diff
@@ -1,15 +1,26 @@
 <div id="firstrunwizard">
 
+<div class="firstrunwizard-header">
+
 <a id="closeWizard" class="close">
-	<img class="svg" src="<?php print_unescaped(OCP\Util::imagePath('core', 'actions/close.svg')); ?>">
+	<img class="svg" src="<?php print_unescaped(OCP\Util::imagePath('core', 'actions/view-close.svg')); ?>">
 </a>
+<div class="logo">
+	<p class="hidden-visually">
+		<?php p($theme->getName()); ?>
+	</p>
+</div>
 <h1><?php p($l->t('A safe home for all your data')); ?></h1>
```

Comment:
Should be themable as well? I'd do the following:

1. Remove the if in the line below and just keep `Access & share`.
2. Use `$theme->getSlogan()` here.

Then the slogan from the theming section is also taken. The hack below was only for the Enterprise Edition for ownCloud.